polaris-icons/icons/StatusActiveMajor.yml

```
@@ -0,0 +1,12 @@
+name: active status
+set: major
+description: Used to represent a resource that is in an Active state. 
+keywords:
+  - active
+  - tick
```

```suggestion
  - tick
  - good
  - positive
  - checkmark
  - circle
  - round
  - approve
```

Optional keywords if you want to use them